File: str_replace/str_replace.js
// ATOMIC_FUNCTION:  replaceAll(old,new)
// TAGS: string
// LANG:  node.js
// FUNCTION: replaceAll(old,new)

var start_val = "<body text='%body%'>";
var expected_end_val = "<body text='black'>";
var end_val = start_val.replace("%body%", "black");

// TEST
if(end_val==expected_end_val){
    console.log('PASS');
}else{
    console.log('FAIL');
}

File: str_replace/str_replace.py
# ATOMIC_FUNCTION:  str.replace(old, new[, max])
# TAGS: string
# LANG:  python
# FUNCTION: str.replace(old, new[, max])

start_val = "<body text='%body%'>"
expected_end_val = "<body text='black'>"

end_val = start_val.replace("%body%","black")

# TEST
if(end_val == expected_end_val):
    print("PASS")
else:
    print("FAIL")






